src/GitVersion.Core.Tests/IntegrationTests/JustSomeTestScenarios.cs

```diff
@@ -0,0 +1,228 @@
+using GitTools.Testing;
+using GitVersion.Core.Tests.Helpers;
+using GitVersion.Model.Configuration;
+using GitVersion.VersionCalculation;
+using NUnit.Framework;
+
+namespace GitVersion.Core.Tests.IntegrationTests;
+
+[TestFixture]
+public class JustSomeTestScenarios : TestBase
+{
+    [Test]
+    public void __Just_A_Test_1__()
```

Comment:
We need a more descriptive name than `__Just_A_Test_1__`. :)

Reply:
I fully agree with you. I need to rename it and give the scenarios a good name. It's pending